src/server/services/toeristische-verhuur.ts

```diff
@@ -172,6 +174,32 @@ export function transformVergunningenToVerhuur(
       status = vergunning.decision;
     }
 
+    if (vergunning.caseType === CaseType.VakantieVerhuur) {
+      vergunning.vergunningId = vergunningen.find((v) => {
+        return (
+          v.caseType === CaseType.VakantieverhuurVergunningaanvraag &&
+          v.location === vergunning.location &&
+          v.dateStart &&
+          vergunning.dateStart &&
+          new Date(vergunning.dateStart) >= new Date(v.dateStart) &&
+          v.dateEnd &&
+          vergunning.dateEnd &&
+          new Date(vergunning.dateEnd) <= new Date(v.dateEnd) &&
+          v.id !== vergunning.id
+        );
+      })?.id;
```

Comment:
Deze logica is beter op zijn plek in de koppel-api denk ik. Kun je dat naar de decos api verplaatsen en er een comment bijzetten over de redenen waarom een vergunningId gemaakt/gekoppeld moet worden?